rocalution/PKGBUILD

```
@@ -11,14 +11,17 @@ depends=('hip' 'rocsparse' 'rocblas' 'rocprim' 'rocrand' 'openmp')
 makedepends=('cmake' 'rocm-cmake' 'git')
 _git='https://github.com/ROCmSoftwarePlatform/rocALUTION'
 source=("$pkgname-$pkgver.tar.gz::$_git/archive/rocm-$pkgver.tar.gz"
-        "rocblas-rocsparse-include-path.patch::$_git/commit/8264818ab790c48f12df45e6dc90d504be72d690.patch")
+        "rocblas-rocsparse-include-path.patch::$_git/commit/8264818ab790c48f12df45e6dc90d504be72d690.patch"
+        "gfx1031.patch")
```

Can you make this patch into an upstream PR? and then download it from the sources array, instead of keeping the source in our repo?

This is so that others in the ROCm community can see and review your contribution besides just keeping it local to archlinux.

---

There's been a merge upstream. So when you update to 5.2.3 it shouldn't be an issue to include this.

```suggestion
        "gfx1031.patch::$_git/commit/9bdff9b0897360a60d21a686f7b988f924aea825")
```

I'll test myself soon. But thought I'd document.